lib/frrscript.h

```diff
@@ -25,7 +25,7 @@
 
 #include <lua.h>
 #include "frrlua.h"
-#include "../bgpd/bgp_script.h"
+#include "bgpd/bgp_script.h"
```

Comment:
Removes ../ include

Reply:
remind me again, what are we pulling in this header for? super weird to pull a daemon header into lib code.

i would say it's worth adding a comment here so someone knows why later looking at it:

```
#include "bgpd/bgp_script.h" /* for BLAH */
```

Reply:
updated https://github.com/FRRouting/frr/pull/8982/commits/8b141afec92b984af8c4b063097d1ce577fda0d8